proxy/proxymanager_test.go

```diff
@@ -349,3 +349,130 @@ func TestProxyManager_StripProfileSlug(t *testing.T) {
 	assert.Equal(t, http.StatusOK, w.Code)
 	assert.Contains(t, w.Body.String(), "ok")
 }
+
+// Test issue #61 `Listing the current list of models and the loaded model.`
+func TestProxyManager_RunningEndpoint(t *testing.T) {
+	// Define a helper struct to parse the JSON response
+	type RunningResponse struct {
+		Running []struct {
+			Model string `json:"model"`
+			State string `json:"state"`
+		} `json:"running"`
+	}
+
+	t.Run("no models loaded", func(t *testing.T) {
+		config := &Config{}
+
+		proxy := New(config)
+		defer proxy.StopProcesses()
```

Comment:
The config, proxy and defer can be shared by all the sub-tests. I suggest defining a single config with models and profiles.